tests/test_bridge_domain.rb

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) 2013-2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/bridge_domain'
+require_relative '../lib/cisco_node_utils/vlan'
+
+include Cisco
+
+# TestBridgeDomain - Minitest for bridge domain class.
+class TestBridgeDomain < CiscoTestCase
+  @@cleaned = false # rubocop:disable Style/ClassVars
+
+  def cleanup
+    BridgeDomain.allbds.each do |_bd, obj|
+      obj.destroy
+    end
+  end
```

Comment:
Please call `super` at the end of your `cleanup` method (same comment in other test file(s)).